cmd/create.go

```diff
@@ -20,18 +20,24 @@ var (
 var componentCreateCmd = &cobra.Command{
 	Use:   "create <component_type> [component_name] [flags]",
 	Short: "Create new component",
-	Long: `Create new component.
-If component name is not provided, component type value will be used for name.
-	`,
-	Example: `  # Create new nodejs component with the source in current directory. 
+	Long: `Create new component to deploy on OpenShift.
+
+If component name is not provided, component type value will be used for the name.
+
+OpenShift built-in types: httpd, nodejs, perl, php, python, ruby, wildfly
```

Comment:
I'm not sure if it is a good idea to have the list here. It will be hard to keep this update.
Just referring to `ocdev component list` is I think enough.